### PR TITLE
Add `--save_nifti` option for `interpret`

### PR DESCRIPTION
**Pull request by camillebrianceau**
I added an option (--save_nifti) to save the interpretation map (individual or groups) in nifty format because it was only saved in PyTorch tensors format.

### CODE REVIEW

File: clinicadl/utils/maps_manager/maps_manager.py [hunk 1]
```diff
@@ -466,12 +488,14 @@ def interpret(
                 label_code=self.label_code,
                 label=self.label,
             )
+            print(data_test)
```
Comment (ravih18) [inline clinicadl/utils/maps_manager/maps_manager.py]: print should be removed

File: clinicadl/utils/maps_manager/maps_manager.py [hunk 2]
```diff
@@ -466,12 +488,14 @@ def interpret(
...
             test_loader = DataLoader(
                 data_test,
                 batch_size=batch_size if batch_size is not None else self.batch_size,
                 shuffle=False,
                 num_workers=n_proc if n_proc is not None else self.n_proc,
             )
+            print(test_loader)
```
Comment (ravih18) [inline clinicadl/utils/maps_manager/maps_manager.py]: print should be removed

File: clinicadl/utils/maps_manager/maps_manager.py [hunk 3]
```diff
@@ -509,6 +533,7 @@ def interpret(
                 for data in test_loader:
                     images = data["image"].to(model.device)
 
+                    # map_pt: np.array
```
Comment (ravih18) [inline clinicadl/utils/maps_manager/maps_manager.py]: should be removed

File: clinicadl/utils/maps_manager/maps_manager.py [hunk 4]
```diff
@@ -522,12 +547,33 @@ def interpret(
                                 f"{self.mode}-{data[f'{self.mode}_id'][i]}_map.pt",
                             )
                             torch.save(map_pt[i], single_path)
+                            if save_nifti:
+                                import nibabel as nib
+                                from numpy import eye
+
+                                single_nifti_path = path.join(
```
Comment (ravih18) [inline clinicadl/utils/maps_manager/maps_manager.py]: to change with pathlib

File: clinicadl/utils/maps_manager/maps_manager.py [hunk 5]
```diff
@@ -522,12 +547,33 @@ def interpret(
...
                         path.join(results_path, f"mean_{self.mode}-{i}_map.pt"),
                     )
+                    if save_nifti:
+                        import nibabel as nib
+                        from numpy import eye
+
+                        output_nii = nib.Nifti1Image(mode_map, eye(4))
+                        nib.save(
+                            output_nii,
+                            path.join(results_path, f"mean_{self.mode}-{i}_map.nii.gz"),
```
Comment (ravih18) [inline clinicadl/utils/maps_manager/maps_manager.py]: to change with pathlib

File: clinicadl/interpret/interpret_cli.py [hunk 6]
```diff
@@ -80,6 +80,13 @@
     default=False,
     help="Overwrite the name if it already exists.",
 )
+@click.option(
```
Comment (ravih18) [inline clinicadl/interpret/interpret_cli.py]: Check if the option cannot be gathered with the one of predict

**Review by ravih18 — CHANGES_REQUESTED** [review]
Lokks good, just a few comment to address.

**Review by mdiazmel — APPROVED** [review]
Looks Ok!

**Review by ravih18 — APPROVED** [review]
Can be merged !

**Pull request merged**